design-proposals/nic-hotplug/nic-hotplug.md

```diff
@@ -392,6 +517,14 @@ performed:
 All these tests have as pre-requirements that the `HotplugInterfaces` feature
 gate is enabled, **and** a secondary network provisioned.
 
+### `virt-launcher` pods network interfaces naming
+* VMs that were created with secondary interfaces can be migrated by 
+KubeVirt upgrade (when workload-strategy is set to Migrate), and after 
+Kubevirt upgrade following user request.
+
+It should cover changes around virt-launcher pod interface name change and in general cover 
+backward compatibility for changes related to the networking code.
```

Comment:
I'm afraid I don't follow this section. Why is this in the middle of a section about tests ?

Reply:
This section describes how the pod interface nameing change should be tested, its a sub-section of `Functional Testing Approach`

Reply:
`Functional Testing Approach` lists in its own heading the KubeVirt required tests (well ... an approximation of the tests).

Then it has a dedicated sub-heading about what tests must be made on the components that are not KubeVirt - i.e.  multus dynamic controller

I find it confusing to see a new sub-heading added pointing at an existing component. 

But honestly I don't have a better suggestion. It feels extra-weird since it feels you're adding design info for something that has already happened :) IMO, there's no point "designing" tests that already exist. 

Feel free to keep this if you want; IMO it's strange, but, I can live with it.

Reply:
I can remove the sub-heading, will that make sense to you?

Reply:
related to https://github.com/kubevirt/community/pull/217#discussion_r1183688449